ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa-4.12-nightly-x86.yaml

```diff
@@ -33,7 +33,7 @@ tests:
       OPENSHIFT_VERSION: "4.12"
       REPLICAS: "24"
     test:
-    - ref: openshift-qe-cluster-density-v2
+    - chain: openshift-qe-control-plane
```

Comment:
There's a node-density test in a below section that I think should be deleted because it'll be incorporated in this test

Reply:
Right the corresponding node-density job is not a cron job per se.
Will remove the reference to it.

Reply:
Updated